js/src/components/paid-ads/audience-section.js

```diff
@@ -2,25 +2,71 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { SelectControl } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
+import useCountryKeyNameMap from '.~/hooks/useCountryKeyNameMap';
 import Section from '.~/wcdl/section';
 import AudienceCountrySelect from '.~/components/audience-country-select';
 
+function toCountryOptions( countryCodes, countryNameMap ) {
+	return countryCodes.map( ( code ) => ( {
+		label: countryNameMap[ code ],
+		value: code,
+	} ) );
+}
+
+/**
+ * Renders <Section> and <Section.Card> UI with country(s) selector.
+ *
+ * @param {Object} props React props.
+ * @param {Object} props.formProps Form props forwarded from `Form` component.
+ * @param {boolean} [props.multiple=true] Whether the selector is multi-selected.
+ * @param {boolean} [props.disabled=false] Whether the selector is disabled.
+ * @param {JSX.Element} [props.countrySelectHelperText] Helper text to be displayed under the selector.
+ */
 const AudienceSection = ( props ) => {
 	const {
 		formProps: { getInputProps },
+		multiple = true,
 		disabled = false,
 		countrySelectHelperText,
 	} = props;
 
+	const countryNameMap = useCountryKeyNameMap();
 	const inputProps = getInputProps( 'country' );
+
+	/**
+	 * Hack to prevent tags being removed even when disabled.
+	 * This could be removed after replacing <AudienceCountrySelect> with <TreeSelectControl>.
+	 */
 	if ( disabled ) {
 		inputProps.onChange = undefined;
 	}
 
+	const selector = multiple ? (
+		<AudienceCountrySelect
+			label={ __( 'Select countries', 'google-listings-and-ads' ) }
+			helperText={ countrySelectHelperText }
```

Comment:
💅  ❔  Related to `countrySelectHelperText`, we don't need it as props, we can have the text inline here, like: 

```suggestion
			helperText={ __(
				'Once a campaign has been created, you cannot change the target country(s).',
				'google-listings-and-ads'
		  ) }
```

Reply:
`<AudienceSection>` is also used in campaign creation and this helper is only shown when editing. So I thought it should be a bit better rather than passing a `@param {'create'|'edit'} mode` for `<AudienceSection>` to control this.

Reply:
> `<AudienceSection>` is also used in campaign creation and this helper is only shown when editing.

D'oh, somehow I missed that. 🤦 

So now, this is my thinking:

- Campaign create: section is enabled, there is no helper text.
- Campaign edit: section is disabled, we show helper text to explain that they can't change the country.

So from the above, the helper text could depend on the `disabled` prop and we don't need `countrySelectHelperText`? 🤔 

```suggestion
			helperText={
				disabled &&
				__(
					'Once a campaign has been created, you cannot change the target country(s).',
					'google-listings-and-ads'
				)
			}
```

Reply:
Thanks for the suggestion! This does give the expected results without adding additional prop. 👍 This does give the expected results without an additional prop.

However, when determined by `disabled`, this implicitly equates disabled logically with creation and edit. While the current result will be correct, it also implicitly reduces maintainability, e.g. if later need to disable the selector control when creating, then `disabled` may not be applicable.

On the other hand, when TreeSelectControl is integrated later, there is still a sentence similar to helper text that needs to be displayed. Looks like they are both are a kind of helper text used in the same meaning though the positions are different. The `countrySelectHelperText` may still be used later, depending on the design of TreeSelectControl and subsequent integration considerations.

![image](https://user-images.githubusercontent.com/17420811/155642917-47b84c40-e3b3-4c46-9f1c-3954d78c89c2.png)

Reply:
Alright, let's go with what we have.